bin/wp-cli.php

```diff
@@ -105,6 +136,12 @@ public function flush() {
 	 * : Which page to start on. This is helpful if you encountered an error on
 	 * page 145/150 or if you want to have multiple processes running at once
 	 *
+	 * [--from=<date>]
+	 * : Index posts published on or after this date.
+	 *
+	 * [--to=<date>]
+	 * : Index posts published on or before this date.
```

Comment:
Specify the date format in these. These are parsed and output with `wp help`, so we want them to be as helpful as possible. Also, I made an earlier comment asking if the dates are inclusive. Looks like you specify that here, so disregard that!